<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSystem</class>
 <widget class="QDialog" name="EditSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>系统配置信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_SystemName">
        <property name="text">
         <string>系统名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_SystemName"/>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>246</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>网络配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_ListenPort">
           <property name="text">
            <string>本地端口</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_ListenPort"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_TargetIP">
           <property name="text">
            <string>目标地址</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_TargetIP"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_TargetPort">
           <property name="text">
            <string>目标端口</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_TargetPort"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>245</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>接收参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_RecvName">
        <property name="text">
         <string>接收参数名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_RecvName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_RecvType">
        <property name="text">
         <string>参数类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_RecvType">
        <item>
         <property name="text">
          <string>short</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>int</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>long</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>float</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>double</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bool</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushBtn_AddRecv">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>144</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushBtn_DeleteRecv">
        <property name="text">
         <string>删除选中参数</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget_Recv"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>发送参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_SendName">
        <property name="text">
         <string>发送参数名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_SendName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_SourceSystem">
        <property name="text">
         <string>数据来源系统</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_SourceSystem"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_SourceData">
        <property name="text">
         <string>对应参数</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox_SourceData"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushBtn_AddSend">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>143</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushBtn_DeleteSend">
        <property name="text">
         <string>删除选中参数</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidget_Send"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_Confirm">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/img/images/704.bmp</normaloff>:/img/images/704.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtn_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/img/images/706.bmp</normaloff>:/img/images/706.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
